packages/amplication-data-service-generator/src/server/message-broker/generate-message-broker-client-options/generate-message-broker-client-options.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  CreateMessageBrokerClientOptionsFactoryParams,
+  EventNames,
+  Module,
+} from "@amplication/code-gen-types";
+import pluginWrapper from "../../../plugin-wrapper";
+
+export async function createGenerateMessageBrokerClientOptionsFunction(
+  eventParams: CreateMessageBrokerClientOptionsFactoryParams["before"]
+): Promise<Module[]> {
+  return pluginWrapper(
+    createGenerateMessageBrokerClientOptionsFunctionInternal,
+    EventNames.CreateMessageBrokerClientOptionsFactory,
+    eventParams
+  );
+}
+
+export async function createGenerateMessageBrokerClientOptionsFunctionInternal(
+  eventParams: CreateMessageBrokerClientOptionsFactoryParams["before"]
+): Promise<Module[]> {
+  return [];
+}
```

Comment:
Remove this function.

knowing that we may use a lot of skeleton events that will only be handled by plugins with no default behavior, we should create an easier and more readable code for these cases. 


for now I suggested using ()=>[] for an empty function, but maybe we should allow null for the second parameter of the PluginWrapper 

@barshimi what do you think?